cfgov/privacy/forms.py

```diff
@@ -164,7 +164,7 @@ def send_email(self):
             body=self.email_body(),
             from_email=settings.DEFAULT_FROM_EMAIL,
             to=[settings.PRIVACY_EMAIL_TARGET],
-            reply_to=[self.cleaned_data["requestor_email"]],
+            reply_to=[self.cleaned_data.get("requestor_email")],
```

Comment:
In this case I believe we should always have `requestor_email` because it is a required field.

```suggestion
            reply_to=[self.cleaned_data["requestor_email"]],
```